ImageNet/inference/Alipay_resnet101_npu_tensorflow.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "v1.0",
+    "author": "Ai CognitiveComputing team in Alipay Group",
+    "authorEmail": "xing.rao@alibaba-inc.com, weihong.wh@alibaba-inc.com",
+    "framework": "tensorflow+NpuInference",
+    "codeURL": "https://github.com/alipay-cognition/DAWNBench_Inference",
+    "commitHash": "",
+    "model": "ResNet101",
+    "hardware": "Alibaba Cloud Npu, from https://promotion.aliyun.com/ntms/act/alinpu.html?spm=5176.12901015.0.i12901015.4c27525cDE1r6q",
```

Comment:
This is meant to be a short name for the hardware. Please remove the URL. If you want to include it, add it under the `misc` field.